src/onelogin/saml2/auth.py

```diff
@@ -111,7 +111,7 @@ def process_response(self, request_id=None):
                 OneLogin_Saml2_Error.SAML_RESPONSE_NOT_FOUND
             )
 
-    def process_slo(self, keep_local_session=False, request_id=None, delete_session_cb=None):
+    def     process_slo(self, keep_local_session=False, request_id=None, delete_session_cb=None, method_data='get_data'):
```

Comment:
I don't think you want the extra whitespace in between the `def` and function name `process_slo`.